.github/workflows/docker.yml

```diff
@@ -36,26 +36,43 @@ jobs:
     steps:
     - name: Checkout
       uses: actions/checkout@v3
-    - name: Prepare tags
-      id: prep
+    - name: Prepare base version
+      id: prep_base_version
       shell: python
       run: |
```

Comment:
This script was already big, but it's huge now. Is there a simple way to have it as a file?

Reply:
Yes, but I decided to do it in a different PR that is still WIP: https://github.com/HathorNetwork/hathor-core/pull/548

Otherwise I think it would become harder for the reviewers to know what changed exactly in the logic